### ISSUE TITLE
When automatic round is off, initializing status is blocking alignment settings

### ISSUE THREAD

**Issue by Apfelstruhdl** [jomjol/AI-on-the-edge-device#3209]
### The Problem

I currently use a short round time and when I want updating the reference information, the round processing blocks the UI.

So I switched the automatic round off. Then the status seems to be stuck with *initializing" and the reference UI is also blocked.

### Version

Rolling

### Logfile

```shell
None
```

### Expected Behavior

Beyond the fix, it might be good to have the option to temporarily pause the round when updatingnreference image taking and alignment settings.

**Comment by caco3** [comment]
> I want updating the reference information, the round processing blocks the UI.

Yes, this is intended. After the ongoing round completes, the reference image task gets run.
How it might be that you are on an edge case where the rounds are tight after each other. if so, please increase the interval.

> So I switched the automatic round off.
As you can see in the documentation this is an expert option only used for debugging. Don't use it as it does not behave as intended.
